components/tabcordion/src/Tabcordion.js

```diff
@@ -1,9 +1,10 @@
 import React, { Children, useEffect, useRef, useState } from 'react';
 import PropTypes from 'prop-types';
+import { useBrand, merge } from '@westpac/core';
 import { useContainerQuery } from '@westpac/hooks';
-
-import { TabItem, TabRow } from './styled';
+import { TabItem, TabRow, AccordionLabel, Panel } from './styled';
```

Comment:
We need to talk about `styled.js`.

Reply:
Are we moving all styled components into the component file that requires them?

Reply:
We should chat about this. I personally like this more :)